### PR TITLE
added HC CUCEK website

### PR DESCRIPTION
**Pull request by rafitc**

### CODE REVIEW

**Comment by Farzanfa** [comment]
All records must be in alphabetical order

**Comment by Farzanfa** [comment]
````
ctrlalttec:
- ttl: 3600
  type: TXT
  value: v=spf1 include:_spf.google.com ~all
- ttl: 3600
  type: MX
  values:
  - exchange: aspmx.l.google.com.
    preference: 1
  - exchange: alt1.aspmx.l.google.com.
    preference: 5
  - exchange: alt2.aspmx.l.google.com.
    preference: 5
  - exchange: alt3.aspmx.l.google.com.
    preference: 10
  - exchange: alt4.aspmx.l.google.com.
    preference: 10

<here>

customizer:
- ttl: 1
  type: CNAME
  value: jasonappah.github.io.
````

File: hackclub.com.yaml [hunk 1]
```diff
@@ -921,3 +921,8 @@ zapdos.servers:
 - ttl: 1
   type: A
   value: 45.79.138.244
+cucek:
+- ttl: 1
+  type: CNAME
+  value: hccucek.netlify.app
```
Comment (lachlanjc) [inline hackclub.com.yaml:927]:
```suggestion
  value: hccucek.netlify.app.
```

**Review by lachlanjc — CHANGES_REQUESTED** [review]
Hey! The subdomains in this file must be sorted alphabetically, so this entry needs to be slotted into the right slot closer to the top for it to work correctly.

**Comment by rafitc** [comment]
@lachlanjc and @Farzanfa  Thanks :)

**Pull request merged**